File: examples/python/transcription-sample.py
from openai import OpenAI
client = OpenAI(api_key="dpais", base_url="http://localhost:8553/v1/openai")

audio_file = open("speech.mp3", "rb")
transcript = client.audio.translations.create(
    model="whisper",
    file=audio_file
)

print(transcript.text)

File: examples/js/transcription-sample.js
import fs from "fs";
import OpenAI from "openai";
import fs from 'fs';


const client = new OpenAI({
  apiKey: 'dpais',
  baseURL: 'http://localhost:8553/v1/openai',
});

async function main() {
    const translation = await openai.audio.translations.create({
        file: fs.createReadStream("speech.mp3"),
        model: "whisper",
    });

    console.log(translation.text);
}
main();